community-operators/cockroachdb/cockroachdb.v2.0.9.clusterserviceversion.yaml

```diff
@@ -0,0 +1,145 @@
+    categories: Database
+    description: CockroachDB
+    containerImage: quay.io/helmoperators/cockroachdb:2.0.9
+    createdAt: 2019-01-24T15-33-43Z
+    support: a-robinson
+  name: cockroachdb.v2.0.9
+  namespace: placeholder
+spec:
+  customresourcedefinitions:
+    owned:
+    - description: CockroachDB
+      displayName: cockroachdb
+      kind: Cockroachdb
+      name: cockroachdbs.charts.helm.k8s.io
+      version: v1alpha1
+  keywords: ['cockroach', 'cockroachdb', '']
+  displayName: cockroachdb
+  provider:
+    name: Helm Community
+  maturity: stable
+  version: 2.0.9
+  links:
+  - name: Helm Chart Source
+    url: https://github.com/cockroachdb/cockroach
+  - name: Configuration Options
+    url: https://github.com/helm/charts/tree/master/stable/cockroachdb#configuration
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  maintainers:
+  - email: alex@cockroachlabs.com
+    name: a-robinson
+  selector:
+    matchLabels:
+      alm-owner-etcd: couchbaseoperator
+      operated-by: couchbaseoperator
+  labels:
+    alm-owner-etcd: couchbaseoperator
+    operated-by: couchbaseoperator
+  install:
+    spec:
+      deployments:
+      - name: cockroachdb-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: cockroachdb
+          template:
+            metadata:
+              labels:
+                name: cockroachdb
+            spec:
+              serviceAccountName: cockroachdb-operator
+              containers:
+              - name: cockroachdb
+                image: quay.io/helmoperators/cockroachdb:2.0.9
+                ports:
+                - containerPort: 60000
+                  name: metrics
+                imagePullPolicy: Always
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: OPERATOR_NAME
+                  value: cockroachdb
+      clusterPermissions:
+      - rules:
+        - verbs:
+          - use
```

Comment:
According to OLM's catalog-operator this is an invalid verb:
```
spec.install.spec.permissions.rules.verbs in body should be one of [*
        assign get list watch create update patch delete deletecollection
        initialize]
```

Reply:
Can confirm that testing this on the 0.12.0 installer does not give the above error.

Reply:
This bug is fixed and merge on the OLM side.